extensions/amp-story/0.1/amp-story.js

```diff
@@ -329,6 +329,17 @@ export class AmpStory extends AMP.BaseElement {
   }
 
   /** @private */
+  addButtonListeners_() {
```

Comment:
Let's remove this method and move the code into `buildButtons_()`. I don't think there's a case where we want to do one without the other.

Reply:
I feel putting it in same function kinda makes unit testing a little tougher

Reply:
You can test the behavior of `buildButtons_()` instead.

Reply:
yep, but if that same function creates the button and also adds the listener there's nothing to stub as I am doing right now.

Reply:
```js
it('adds event listener for buttons', () => {
  story.buildButtons_();
  const nextStub = sandbox.stub(story, 'next_');
  this.nextButton_.dispatchEvent(new Event('click'));
  expect(nextStub).calledOnce;
});
```

Reply:
Done